editor/editor_command_palette.cpp

```diff
@@ -264,7 +276,7 @@ EditorCommandPalette::EditorCommandPalette() {
 	add_child(vbc);
 
 	command_search_box = memnew(LineEdit);
-	command_search_box->set_placeholder("search for a command");
```

Comment:
It would be better to change this to "Filter commands".

Reply:
sure.